src/MaliitPluginsConfig.cmake.in

```diff
@@ -1,7 +1,7 @@
 include(CMakeFindDependencyMacro)
 
-find_dependency(Qt5Core @Qt5Core_VERSION@)
-find_dependency(Qt5Gui @Qt5Gui_VERSION@)
-find_dependency(Qt5Quick @Qt5Quick_VERSION@)
+find_dependency(Qt6Core @Qt6Core_VERSION@)
```

Comment:
Just bumping the version smell like breaking Qt5 support. Right?

Reply:
Oh, right, I fixed it to support both versions now.